typescript/cli/src/config/warp.ts

```diff
@@ -274,12 +274,15 @@ export async function createWarpRouteDeployConfig({
         context.multiProvider,
         warpRouteDeployConfig,
       );
+
+      const symbol = tokenMetadata.getSymbol('');
       assert(
-        tokenMetadata?.symbol,
-        'Error deriving token metadata, please check the provided token addresses',
+        symbol === undefined,
```

Comment:
i think the assertion here should be `!symbol` because if the assertion is true then no error will be thrown but you want to throw an error if symbol is undefined that is probably why you alsos neded to add the `!` operator below